src/main/java/frc/robot/commands/SnapAndShootCommand.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.subsystems.DriveSubsystem;
+import frc.robot.subsystems.ElevatorSubsystem;
+import frc.robot.subsystems.ShooterSubsystem;
+import frc.robot.subsystems.VisionSubsystem;
+
+/**
+ *
+ * @author aryan
+ */
+public class SnapAndShootCommand extends ParallelCommandGroup{
```

Comment:
i really like that its easy to decompose our commands like this. 
i really dislike the way wpilib requires a subclass just to make a list of commands. I wonder if we could make a simple class that makes it easy to make a parallel or serial command without needing a new class.

Reply:
Yeah that is exactly what I was thinking. After doing some reading I came across the fact that we can pass in an array of commands and it creates a group from that.